src/constants/index.ts

```diff
@@ -51,6 +52,7 @@ export const INITIAL_FORM_CONFIG: IFormConfigurator = Object.freeze({
     initialOutputMint: WRAPPED_SOL_MINT.toString(),
   },
   useUserSlippage: true,
+  refetchIntervalForTokenAccounts: 10000,
```

Comment:
_app.tsx and FormConfigurator are intended for Terminal preview site.
you should have 10_000 passed down from somewhere, or directly in the accounts.tsx contexts so the default behaviour is kept

Reply:
removed from form